platform/src/test/java/platformstepdefinition/PF_AccountAndSecurity.java

```diff
@@ -63,7 +63,6 @@ public void Account_Information_Cancel_Button() throws Throwable {
 	public void Validate_Missing_Values_Of_Account_Information() throws Throwable {
 		AccountAndSecurity MissingValues = new AccountAndSecurity(driver);
 		MissingValues.MissngValuesAccountAndSecurityInformation();
```

Comment:
Please refactor method to verbNoun format.
Cant determine what this method does

Reply:
Resolved

Reply:
No changes observed in code.
Please start method name with verb